en/mulesoft/modules/ROOT/pages/ms_composer_salesforce_reference.adoc

```diff
@@ -89,6 +89,18 @@ Checks for soft-deleted records. Soft-deleted records are stored in Salesforce f
 Records are typically stored for approximately 15 days before being hard-deleted and irretrievable.
 * You can't modify the soft-delete timestamp.
 
+=== Trigger Conditions
+
+You can add trigger conditions to specify which conditions should trigger a workflow. For example, you can restrict a workflow to trigger only when a specific field meets the conditions you specify.
```

Comment:
Kim didn't like it when I started sentences with "For example," but since I was a regular offender, I don't have a suggestion other than making it a comma instead like: 
You can add trigger conditions to specify which conditions will trigger a workflow, for example, ...